pandas/core/generic.py

```diff
@@ -9420,7 +9420,12 @@ def _doc_parms(cls):
 _any_see_also = """\
 See Also
 --------
-pandas.DataFrame.all : Return whether all elements are True.
+any : Standard library version of this method.
```

Comment:
Can't recall any other instance where we've listed a standard library method like this before; I'd remove as this is about as implicit as it gets

Reply:
There are a few. Was at a Python Sprint in London and the organiser suggested it. But I'll remove. Thank you!

Reply:
Hmm OK. Well if that organizer was @datapythonista then his word trumps mine :-)

Reply:
It was! Should I revert back to how it was?

Reply:
I'd still keep it as is because its trivial either way so the least amount of effort at this point (i.e. doing nothing) I think is the best course of action. Fine if others disagree

Reply:
Not really sure if we've got a discussion about it. But yes, there are some other methods like `str.split` where equivalent standard library were listed.

I'm happy having or not having them. I guess some user new to Python could be happy to discover that Python itself does it. And I don't think the `See Also` is never that long. But happy if we don't include them.